### PR TITLE
feat(aom): support aom multi account aggregation rule

### PR DESCRIPTION
**Pull request by saf3dfsa**
**What this PR does / why we need it**:
Support aom multi account aggregation rule management.

## PR Checklist

* [x] Tests added/passed.

```
go test ./huaweicloud/services/acceptance/aom -v -run TestAccMultiAccountAggregationRule_basic -timeout 360m -parallel 4            
=== RUN   TestAccMultiAccountAggregationRule_basic
=== PAUSE TestAccMultiAccountAggregationRule_basic
=== CONT  TestAccMultiAccountAggregationRule_basic
--- PASS: TestAccMultiAccountAggregationRule_basic (15.22s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/aom       15.871s

 go test ./huaweicloud/services/acceptance/aom -v -run TestAccDataSourceMultiAccountAggregationRules_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceMultiAccountAggregationRules_basic
=== PAUSE TestAccDataSourceMultiAccountAggregationRules_basic
=== CONT  TestAccDataSourceMultiAccountAggregationRules_basic
--- PASS: TestAccDataSourceMultiAccountAggregationRules_basic (8.50s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/aom       9.098s
```

* [x] Documentation updated.
* [x] Schema updated.
* [x] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

**Comment by qigaoneng** [comment]
OK

**Comment by Jason-Zhang9309** [comment]
put your test results here

File: docs/resources/aom_multi_account_aggregation_rule.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+<a name="accounts_struct"></a>
+The `accounts` block supports:
+
+* `id` - (Required, String) Specifies the account ID.
+
+* `name` - (Required, String) Specifies the account name.
+
+* `urn` - (Optional, String) Specifies the uniform resource name of the account.
+
+* `join_method` - (Optional, String) Specifies the time how the account joined in the organization.
```
Comment (Jason-Zhang9309) [inline docs/resources/aom_multi_account_aggregation_rule.md]:
```suggestion
* `join_method` - (Optional, String) Specifies the method how the account joined in the organization.
```

File: docs/data-sources/aom_multi_account_aggregation_rules.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<a name="accounts_struct"></a>
+The `accounts` block supports:
+
+* `id` - Indicates the account ID.
+
+* `name` - Indicates the account name.
+
+* `urn` - Indicates the uniform resource name of the account.
+
+* `join_method` - Indicates the time how the account joined in the organization.
```
Comment (Jason-Zhang9309) [inline docs/data-sources/aom_multi_account_aggregation_rules.md]:
```suggestion
* `join_method` - Indicates the method how the account joined in the organization.
```

File: docs/resources/aom_multi_account_aggregation_rule.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+* `id` - Indicates the resource ID which is same as `instance_id`.
+
+## Import
+
+The AOM multi account aggregation rule resource can be imported using the `id`, e.g.
+
+```bash
+$ terraform import huaweicloud_aom_multi_account_aggregation_rule.test <id>
+```
```
Comment (Jason-Zhang9309) [inline docs/resources/aom_multi_account_aggregation_rule.md:91]: enterprise_project_id can't be imported, should add a description here

File: huaweicloud/services/acceptance/aom/resource_huaweicloud_aom_multi_account_aggregation_rule_test.go [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+// lintignore:AT004
+func testMultiAccountAggregationRuleBase(name string) string {
+	return fmt.Sprintf(`
+provider "huaweicloud" {
+  endpoints = {
+    aom = "https://aomperform.cn-north-7.myhuaweicloud.com/"
+  }
+}
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/aom/resource_huaweicloud_aom_multi_account_aggregation_rule_test.go]:
```suggestion
```

File: huaweicloud/services/aom/resource_huaweicloud_aom_multi_account_aggregation_rule.go [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+	d.SetId(d.Get("instance_id").(string))
+
+	return resourceMultiAccountAggregationRuleRead(ctx, d, meta)
+}
+
+func buildMultiAccountAggregationRuleBodyParams(d *schema.ResourceData) map[string]interface{} {
+	bodyParams := map[string]interface{}{
+		"dest_prometheus_id": d.Get("instance_id"),
+		"accounts": utils.ValueIgnoreEmpty(buildMultiAccountAggregationRuleAccounts(
+			d.Get("accounts").(*schema.Set).List())),
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/aom/resource_huaweicloud_aom_multi_account_aggregation_rule.go]: buildMultiAccountAggregationRuleAccounts should return nil if no accounts specified
```suggestion
		"accounts": buildMultiAccountAggregationRuleAccounts(
			d.Get("accounts").(*schema.Set).List()),
```

File: huaweicloud/services/aom/resource_huaweicloud_aom_multi_account_aggregation_rule.go [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+		"send_to_source_account": utils.ValueIgnoreEmpty(d.Get("send_to_source_account")),
+	}
+
+	if v, ok := d.GetOk("services"); ok {
+		bodyParams["metrics"] = buildMultiAccountAggregationRuleMetrics(v.(*schema.Set).List())
+	}
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/aom/resource_huaweicloud_aom_multi_account_aggregation_rule.go]: why not put this part in the bodyParams definiton

**Pull request merged**